lib/genres.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'text_styles.dart';
+import 'movie.dart';
+
+class WidgetedGenres extends StatelessWidget {
+  final Movie movie;
+
+  const WidgetedGenres(
+    this.movie, {
+    super.key,
+  });
+
+  static const genresWordPadding = EdgeInsets.only(left: 20);
+
+  List<Widget> _createWidgets() {
+    return movie.genres
+        .map(
+          (e) => Padding(
+            padding: genresWordPadding,
```

Comment:
Instead to do this you can use EdgeInsets.only(left: paddingLeft), paddingLeft is a new const that you need to create

Reply:
Done